doc/tutorials/BasicApps.hs

```diff
@@ -41,7 +41,7 @@ data SplitData =
     deriving stock (Show, Generic)
 
 -- For a 'real' application use 'makeIsDataIndexed' to ensure the output is stable over time
-PlutusTx.unstableMakeIsData ''SplitData
+PlutusTx.makeIsData ''SplitData
```

Comment:
`makeIsData` doesn't exist anymore, it's either `unstableMakeIsData` or `makeIsDataIndexed`, as the comment says.

Reply:
Hmm... This doesn't reflect what I've been experiencing on https://playground.plutus.iohkdev.io/ (which seems to be the advised way to learn this kind of thing).

Reply:
the playground isn't running the latest version from master, it is only updated occasionally.